draft-ietf-quic-load-balancers.md

```diff
@@ -526,30 +526,33 @@ trusted network service. One requires no shared state; the server need only be
 configured to trust a retry service, though this imposes other operational
 constraints. The other requires shared key, but has no such constraints.
 
-Retry services MUST forward all non-Initial QUIC packets, as well as Initial
-packets from the server.
-
 ## Common Requirements
 
 Regardless of mechanism, a retry service has an active mode, where it is
 generating Retry packets, and an inactive mode, where it is not, based on its
 assessment of server load and the likelihood an attack is underway. The choice
-of mode MAY be made on a per-packet basis, through a stochastic process or based
-on client address.
+of mode MAY be made on a per-packet or per-connection basis, through a
+stochastic process or based on client address.
 
 A retry service MUST forward all packets for a QUIC version it does not
 understand. Note that if servers support versions the retry service does not,
-this may unacceptably increase loads on the servers. However, dropping these
-packets would introduce chokepoints to block deployment of new QUIC versions.
-Note that future versions of QUIC might not have Retry packets, or require
-different information.
+this may increase loads on the servers. However, dropping these packets would
```

Comment:
Maybe something more like "may lead to increased load on the server".

Reply:
I prefer the more concise sentence.